src/development.ts

```diff
@@ -10,7 +10,9 @@ function inIframe() {
   }
 }
 
-const IS_BROWSER = typeof window !== 'undefined'
+const IS_BROWSER = 
+  typeof window !== 'undefined' && 
+  window.navigator.product === 'Gecko'
```

Comment:
Maybe just filtering out React Native?
```suggestion
  window.navigator.product !== 'ReactNative'
```

Reply:
It could be. First, I wrote like that too then I changed it to gecko for `IS_BROWSER` context.

Reply:
Makes sense!